docs/_docs/Starters Academy (LDSSA)/01-Starters-Academy-(Course).md

```diff
@@ -5,237 +5,169 @@ order: 1
+As of its seventh edition (2023) it consists of 34 Learning Units, 5 Hackathons and 1 capstone project. It contains 3 units to be done 
+during the admission process, see more about it in the [admission process](../../Applicants/Application-process). This is the fourth edition held remotely.
 
 _The **Data Science Starters Academy** should not be mistaken for the 
 [Lisbon Data Science Academy](../../About us/Lisbon-Data-Science-Academy-(Organization)), which 
 is the organization responsible for running the course._
 
 
-## Structure 
-The Starters Academy lasts approximately 40 weeks of part-time work over the course of 8 months. 
-Of that time, there are 9 days where we request of you to be virtually present (usually on Sundays), 
-and the rest is done remote via GitHub, Slack and the Academy's grading and submission system. 
+## 1. Structure 
+The Starters Academy lasts 34 weeks of part-time work over the course of 8 months. 
+Of that time, there are 8 days where we request you to be virtually present (usually on Sundays).
+The rest is done remotely via GitHub, Slack and the Academy's grading and submission system. 
 *You do not need to quit your primary responsibility to take part in this course.*
 
 The curriculum can be referenced via the [curriculum-development repo](https://github.com/LDSSA/curriculum-development).
 
-### Time commitment
-You should be prepared to spend a recommended *10 hours per week* on the course and an absolute minimum of *5 hours per week* in case you are having a busy week.
+### 1.1 Time commitment
+You should be prepared to spend about *10 hours per week* on the course and an absolute minimum of *5 hours per week* in case you are having a busy week. The weekly time requirements might be higher during specialization 1 and the capstone project.
 
-
-### Introduction session
+### 1.2 Introductory session
 
 The introduction session is a 2-hour session that kicks off the academy. 
 
-In this session all rules and schedule 
-present in this page will be presented, together with any practical details for the current batch. Additionally 
+In this session, all the rules of the Academy
+will be presented, together with any practical details for the current batch. Additionally, 
 we will go over the following topics:
 
-* Code of Conduct
+* Course structure and schedule
+* Code of conduct
 * How to ask for and give help
-* Bootcamp Experience
-* Criteria for Graduation
-* Member Directory
-
-
-### Bootcamp Details
+* Criteria for graduation
 
-The _Bootcamp_ is a 2-day event. The first hackathon is 
-three weeks after the bootcamp.
+### 1.3 Bootcamp Details
 
-During the _Bootcamp_ the students and teachers will participate virtually in a series of 
-presentation and exercises, about 10 hours per day, and the students will go through 14 _Small 
-Learning Units_. 
+The _Bootcamp_ is a 2-day event. In batch7, it will be split in two parts, taking place on two Sunday mornings.
```

Comment:
Specialization 1 is called Bootcamp and binary classification, so the from the I understand that bootcamp are just the two days with the presentations. I wouldn't call it a 3-week event because then also the other specializations are 4 week events. I can also leave out the reference to bootcamp and say that specialisation 1 includes two presentation days.

Reply:
For me, it is just confusing to say it is a 2-day event. Might be clearer to say it is two days with presentations.

Reply:
I changed it to two days with virtual classes.